index.html

```diff
@@ -277,5 +277,69 @@ <h3>Skills</h3>
         </div>
       </div>
     </section>
+    <section id="contact">
+      <div class="description_contact">
+        <p>
+          i'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </div>
+      <form
+        id="contact-form"
+        action="https://formspree.io/f/xayznown"
+        method="POST"
+      >
+        <ul id="form-list">
+          <li>
+            <input
+              type="text"
+              name=""
+              id=""
```

Comment:
- [ ] The form is not getting submitted successfully. I would advise you to fill out the empty attributes as the name attribute is the issue here

![image](https://user-images.githubusercontent.com/86200356/212869889-841fa2a7-0f38-4dfb-89d4-c603fc566a3f.png)

Reply:
Fixed.